upup/models/cloudup/resources/addons/coredns.addons.k8s.io/k8s-1.6.yaml.template

```diff
@@ -111,10 +111,10 @@ spec:
         imagePullPolicy: IfNotPresent
         resources:
           limits:
-            memory: 170Mi
-          requests:
-            cpu: 100m
-            memory: 70Mi
+          memory: {{ KubeDNS.MemoryLimit }}
```

Comment:
I think these new lines are missing a level of indentation

Reply:
Good catch peter. This is fixed.